packages/buidler-core/src/builtin-tasks/flatten.ts

```diff
@@ -66,9 +66,7 @@ export default function() {
       }
 
       const packageJson = await getPackageJson();
-      flattened += `// Sources flattened with buidler v${
-        packageJson.version
-      } https://getbuidler.com`;
+      flattened += `// Sources flattened with buidler v${packageJson.version} https://getbuidler.com`;
```

Comment:
`prettier` was also upgraded. Maybe we should pint it's version? What do you think @jjant ?

Reply:
Yeah, I think we should just pin it down,